### PR TITLE
Generate cluster specs from variables instead of templates.

### PR DESCRIPTION
**Pull request by wandera-petr-cerny**

### CODE REVIEW

**Comment by coufalja** [comment]
👏

File: aws/ig/main.tf [hunk 1]
```diff
@@ -1,9 +1,40 @@
...
+    } 
+    spec = merge({
+      cloudLabels = var.cloud-labels
+      nodeLabels = var.node-labels
+      associatePublicIp = var.visibility == "public"
+      image = var.image
+      machineType = var.type
+      maxSize = var.max-size
+      minSize = var.min-size
+      role = "Node"
```
Comment (wandera-pchung) [inline aws/ig/main.tf:19]: Are we no longer using this as a variable anymore?

**Review by wandera-pchung — APPROVED** [review]

File: aws/ig/main.tf (hunk 1 above)
Comment (wandera-petr-cerny) [reply aws/ig/main.tf:19]: We never did - check the removed `aws/ig/ig-spec-template.tf` above :)

**Review by coufalja — APPROVED** [review]

File: aws/ig/main.tf (hunk 1 above)
Comment (gerricchaplin) [reply aws/ig/main.tf:19]: Perhaps he meant the wrong one.

File: aws/mig/main.tf [hunk 2]
```diff
@@ -1,9 +1,47 @@
...
+    }
+    spec = merge({
+      cloudLabels = var.cloud-labels
+      nodeLabels = var.node-labels
+      associatePublicIp = var.visibility == "public"
+      image = var.image
+      machineType = var.type
+      maxSize = var.max-size
+      minSize = var.min-size
+      role = "Node"
```
Comment (gerricchaplin) [inline aws/mig/main.tf:19]: Role was a variable here? https://github.com/wandera/karch/pull/47/files#diff-e7d545d601c7e7198d1556d3d610abeafb31309576e9c03161ea53a2cce5a6a5L18

**Review by gerricchaplin — APPROVED** [review]

**Pull request merged**